File: daily_commit_20210412.c
#include <stdio.h>

void howManySeconds(int hours)
{
    int seconds = hours * 60 * 60;
    printf("%d\n",seconds);
}

int main()
{
    howManySeconds(2);
    howManySeconds(10);
    howManySeconds(24);

}

File: daily_commit_20210412.java
public class Main {

    public static void main(String[] args) {

        int a = howManySeconds(2);
        System.out.println(a);
        int b = howManySeconds(10);
        System.out.println(b);
        int c = howManySeconds(24);
        System.out.println(c);

    }

    private static int howManySeconds(int hours){
        return hours * 60 * 60;
    }
}
